src/test/encoding/types.h

```diff
@@ -192,6 +192,10 @@ TYPE_FEATUREFUL(InodeStore)
 TYPE_FEATUREFUL(MDSMap)
 TYPE_FEATUREFUL(MDSMap::mds_info_t)
 
+#include "mds/FSMap.h"
+//TYPE_FEATUREFUL(Filesystem)
```

Comment:
Yeah, I don't think we need the Filesystem one, it's never encoded on its own anywhere